bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/util/ThingWebClientUtil.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.thing.util;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.thing.ThingUID;
+
+/**
+ * {@link ThingWebClientUtil} provides an utility method to create a valid consumer name for Jetty WEB clients.
```

Comment:
You can use it with any client and it is an implementation detail that Jetty is used... which could be switched one day.

```suggestion
 * {@link ThingWebClientUtil} provides an utility method to create a valid consumer name for web clients.
```

Reply:
Chnaged